lib/transformers/tokend.js

```diff
@@ -29,7 +30,7 @@ function collectTransformables(properties, keyPath) {
     const value = properties[key];
 
     // Don't walk anything that's not an Object. `!= null` covers both null and undefined.
```

Comment:
I don't think this comment applies any more, since we're using `isPlainObject` instead of `source == null`.